File: v2/csharp/src/main/csharp/vestorly/Model/Newsletter.cs
using System;
using System.Text;
using System.Collections;
using System.Collections.Generic;
using System.Runtime.Serialization;

namespace vestorly.model {
  [DataContract]
  public class Newsletter {
    
    
    [DataMember(Name="_id", EmitDefaultValue=false)]
    public string id { get; set; }

    
    
    [DataMember(Name="is_sent", EmitDefaultValue=false)]
    public bool? isSent { get; set; }

    
    
    [DataMember(Name="is_default", EmitDefaultValue=false)]
    public bool? isDefault { get; set; }

    
    
    [DataMember(Name="click_count", EmitDefaultValue=false)]
    public long? clickCount { get; set; }

    
    
    [DataMember(Name="unique_click_count", EmitDefaultValue=false)]
    public long? uniqueClickCount { get; set; }

    
    
    [DataMember(Name="total_click_count", EmitDefaultValue=false)]
    public long? totalClickCount { get; set; }

    
    public override string ToString()  {
      var sb = new StringBuilder();
      sb.Append("class Newsletter {\n");
      
      sb.Append("  id: ").Append(id).Append("\n");
      
      sb.Append("  isSent: ").Append(isSent).Append("\n");
      
      sb.Append("  isDefault: ").Append(isDefault).Append("\n");
      
      sb.Append("  clickCount: ").Append(clickCount).Append("\n");
      
      sb.Append("  uniqueClickCount: ").Append(uniqueClickCount).Append("\n");
      
      sb.Append("  totalClickCount: ").Append(totalClickCount).Append("\n");
      
      sb.Append("}\n");
      return sb.ToString();
    }
  }
  
  
}

File: v2/java/src/main/java/vestorly/client/model/Newsletter.java
package vestorly.client.model;


import com.wordnik.swagger.annotations.*;
import com.fasterxml.jackson.annotation.JsonProperty;


@ApiModel(description = "")
public class Newsletter  {
  
  private String id = null;
  private Boolean isSent = null;
  private Boolean isDefault = null;
  private Long clickCount = null;
  private Long uniqueClickCount = null;
  private Long totalClickCount = null;

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("_id")
  public String getId() {
    return id;
  }
  public void setId(String id) {
    this.id = id;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_sent")
  public Boolean getIsSent() {
    return isSent;
  }
  public void setIsSent(Boolean isSent) {
    this.isSent = isSent;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("is_default")
  public Boolean getIsDefault() {
    return isDefault;
  }
  public void setIsDefault(Boolean isDefault) {
    this.isDefault = isDefault;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("click_count")
  public Long getClickCount() {
    return clickCount;
  }
  public void setClickCount(Long clickCount) {
    this.clickCount = clickCount;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("unique_click_count")
  public Long getUniqueClickCount() {
    return uniqueClickCount;
  }
  public void setUniqueClickCount(Long uniqueClickCount) {
    this.uniqueClickCount = uniqueClickCount;
  }

  
  /**
   **/
  @ApiModelProperty(value = "")
  @JsonProperty("total_click_count")
  public Long getTotalClickCount() {
    return totalClickCount;
  }
  public void setTotalClickCount(Long totalClickCount) {
    this.totalClickCount = totalClickCount;
  }

  

  @Override
  public String toString()  {
    StringBuilder sb = new StringBuilder();
    sb.append("class Newsletter {\n");
    
    sb.append("  id: ").append(id).append("\n");
    sb.append("  isSent: ").append(isSent).append("\n");
    sb.append("  isDefault: ").append(isDefault).append("\n");
    sb.append("  clickCount: ").append(clickCount).append("\n");
    sb.append("  uniqueClickCount: ").append(uniqueClickCount).append("\n");
    sb.append("  totalClickCount: ").append(totalClickCount).append("\n");
    sb.append("}\n");
    return sb.toString();
  }
}
